codes/quantum/qubits/quantum_goppa.yml

```diff
@@ -0,0 +1,117 @@
+product with weights s_i on \( \mathbb{F}_q^n\) is equivalent to C(H). Therefore, the orthogonal code of C(G) is generated by H. Using these properties and the assumption
+that H is a subgroup of G, we can construct a classical Goppa code C(D,G), where D is the sum of all \(P_i\). Using C(D,G), we can construct an binary [[n,k,d]] quantum stabilizer code.'  
+
+protection: 'Protects against weight t errors where \( 0 < t \leq  \lfloor \frac{d^*-g-1}{2} \) where \( d^* = deg G + 2 -2g \) and g is the genus of the function field.'  
+
+# This field starts a list of specific labeled subfields; do not leave it 
+# empty. If empty, comment out. Also, indentations are important!
+features:
+
+  # The fields below relate to specific properties of the code that are worth
+  # noting. In some cases (see below), there might be multiple pieces of
+  # information associated with a field (e.g., mutliple encoders/decoders,
+  # multiple facts about fault tolerance, etc.); in such cases, the field
+  # accepts a list of text entries, with one item per piece of information.
+  #
+  # Any fields below should be omitted if they do not apply.
+
+  # rate: 'k/n'
+
+  # This field that starts a list of strings, indexed by a dash; do not leave it
+  # empty. If empty, comment out.
+  encoders: 
+    - 'Encoding defined in Matsumoto paper \cite{arxiv:quant-ph/0107129}. The Matsumoto encoding uses a technique from Ashikhmin and Knill \cite{arxiv:quant-ph/0005008} to encode
+	  quantum stabilizer codes.'
+
+  #transversal_gates: 'Transversal ... gates \cite{doi:ok-paper}. Comment out if doesn''t apply.'
+
+  #general_gates:
+  #  - 'Universal gate set achieved by either additional ... gate.'
+  #  - 'Magic-state distillation protocols'
+  #  - 'kth Clifford hierarchy gates obtained by ... circuits'
+
+  decoders: 
+   - 'Farran algorithm \cite{arxiv:math.AG/9910151}'
+   
+
+#  fault_tolerance:
+#    - 'Transversal gates are fault-tolerant w.r.t. ... noise \cite{doi:ok-paper}'
+#    - 'Other fault-tolerant gadgets (measurements, encoders, error correcting steps)'
+#    - 'Noise-model-preserving gadgets, noise-biased gates, fault-tolerant flag error correction'
+#    - 'Pieceable fault tolerance.'
+
+#  threshold: 
+#    - '2\% error-correction threshold  ... with perfect ... decoder \cite{arxiv:another-paper}.'
+  
+ # Propose a field that you feel is applicable to a large and interesting class of codes
+#some_other_field:
+ # - 'detail'
+
+#realizations:
+  # List and explain the different "domains" of realizations in list items.
+  # Group closely related realizations into a single item. It's OK to have a
+  # single list item. Each item should more or less correspond to what would be
+  # said in a paragraph.
+  # - 'Code used in DVDs \cite{doi:####...}, 5G, etc.'
+  #- 'Realized in trapped-ion quantum devices \cite{arXiv:####.#####}, etc.'
+notes:
+  - 'k  = \(dim G - dim(G-P_1 - \cdots - P_n - \sigma P_1 - \cdots  - \sigma P_n) - n\)' 
```

Comment:
Move k to end of technical discussion in description field. Move d to protection field. Move CSS plug to a cousin entry for code_id:css.